drafts/str4d-stratum/draft1.rst

```diff
@@ -0,0 +1,511 @@
+Abstract
+========
+
+This ZIP describes the Zcash variant of the Stratum protocol, used by miners to
+communicate with mining pool servers.
+
+This document follows the common conventions defined for ZIPs in [ZIP-1]_.
+
+.. [ZIP-1] https://github.com/zcash/zips/blob/master/zips/0001.rst
+
+
+Motivation
+==========
+
+Many existing cryptocurrency miners and pools use the original Stratum protocol
+[Slushpool-Stratum]_ [Bitcointalk-Stratum]_ for communication, in situations
+where the miner does not require any control over what they mine (for example, a
+miner connected to a local [P2Pool]_ node). However, the protocol is very
+specific to Bitcoin, in that it makes assumptions about the block header format,
+and the available nonce space [Bitcoin-Block]_. Zcash has made changes that
+invalidate these assumptions.
+
+Having a formal specification for a Zcash-compatible Stratum-style mining
+protocol means that existing pool operators and miner authors can quickly and
+easily migrate their frameworks to the Zcash network, with no ambiguity about
+interoperability.
+
+.. [Slushpool-Stratum] Slush Pool. *Stratum Mining Protocol*.
+  URL: https://slushpool.com/help/#!/manual/stratum-protocol
+  (visited on 2016-09-24)
+.. [Bitcointalk-Stratum] -ck, ed. *Stratum protocol documentation*. Bitcoin Forum.
```

Comment:
(and I recall it being like this for me back in NZ too when I was originally writing this spec, so I'm going to avoid adding references to en.bitcoin.it until further notice.)